engine/src/PlatformController.cpp

```diff
@@ -281,4 +280,20 @@ namespace engine::platform {
         core::Controller::get<PlatformController>()->_platform_on_window_close(window);
     }
 
+    void PlatformController::set_cursor_visible(bool flag) {
+        auto platform = engine::core::Controller::get<platform::PlatformController>();
```

Comment:
Pošto se ova funkcija već nalazi unutar instance PlatformController, čini mi se da bi bilo dovoljno koristiti window() umesto platform->window(). To bi učinilo kod čitljivijim i malo pojednostavilo pristup.